models/db/list.go

```diff
@@ -202,3 +202,8 @@ func FindAndCount[T any](ctx context.Context, opts FindOptions) ([]*T, int64, er
 	}
 	return objects, cnt, nil
 }
+
+func Exists[T any](ctx context.Context, opts FindOptions) (bool, error) {
+	var bean T
+	return GetEngine(ctx).Where(opts.ToConds()).Exist(&bean)
```

Comment:
Technically, that's still a possible security issue.
However, if our reviewers review dutifully it will never occur

Reply:
This should be allowed to check the whole table.
This PR just make possible technically error less but it cannot stop all errors. For review dutifully, I think we can have discuss on discord how should we improve our review progress.

Reply:
I updated `Exist` but for `Exists` which like `Find`, it should allow blank condition.